homeassistant/components/launch_library/sensor.py

```diff
@@ -41,12 +47,26 @@
 )
 
 
-SENSOR_DESCRIPTION = SensorEntityDescription(
+NEXT_LAUNCH_SENSOR_DESCRIPTION = SensorEntityDescription(
     key="next_launch",
     icon="mdi:rocket-launch",
     name=DEFAULT_NAME,
 )
 
+LAUNCH_TIME_SENSOR_DESCRIPTION = SensorEntityDescription(
+    key="launch_time",
+    icon="mdi:clock-outline",
+    name="Launch time",
+    device_class=SensorDeviceClass.TIMESTAMP,
+)
+
+LAUNCH_PROBABILITY_SENSOR_DESCRIPTION = SensorEntityDescription(
+    key="launch_probability",
+    icon="mdi:horseshoe",
+    name="Launch Probability",
+    native_unit_of_measurement=PERCENTAGE,
+)
```

Comment:
Make this a tuple with all `SENSOR_DESCRIPTIONS: tuple[SensorEntityDescription, ...] = (SensorEntityDescription(...),)`